src/Compilers/Core/AnalyzerDriver/AnalyzerManager.cs

```diff
@@ -301,6 +301,20 @@ internal bool IsSupportedDiagnostic(DiagnosticAnalyzer analyzer, Diagnostic diag
 
             foreach (var diag in supportedDiagnostics)
             {
+                if (HasNotConfigurableTag(diag.CustomTags))
```

Comment:
Consider creating extension method diag.IsNotConfigurable() (similar to what we use in CSharpFilterDiagnostics.cs)